src/lib.rs

```diff
@@ -1504,6 +1504,16 @@ impl<K: KVStore + Sync + Send + 'static> Node<K> {
 		}
 	}
 
+	/// Update the config for a previously opened channel.
+	pub fn update_channel_config(
+		&self, channel_id: &ChannelId, counterparty_node_id: PublicKey,
+		channel_config: ChannelConfig,
```

Comment:
Maybe take a reference since the method doesn't need ownership? Or is that an issue for bindings?